bazel/api-gen/rendering/templates/header-api.tsx

```diff
@@ -21,6 +21,14 @@ import {DocsPillRow} from './docs-pill-row';
 export function HeaderApi(props: {entry: DocEntryRenderable; showFullDescription?: boolean}) {
   const entry = props.entry;
 
+  // TODO: This link point to the main branch.
+  // When ADEV is not deployed on the main branch branch anymore,
+  // We should update it to point to the tag of the released version which ADEV runs on.
+
+  const sourceUrl = isDocEntryWithSourceInfo(entry)
+    ? `https://github.com/angular/angular/blob/main${entry.source.filePath}#L${entry.source.startLine}-${entry.source.endLine}`
```

Comment:
is `entry.source.filePath` guaranteed to be `/` prefixed?

Reply:
Looking again closer, it looks like the extraction test json file doesn't have it as prefixed, so it wouldn't even point to the correct place there.

Reply:
I just checked against the FW directly, the change is correct, the path has a leading `/`. I changed the test data to reflect that.